guides/source/layout.html.erb

```diff
@@ -1,31 +1,37 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title><%= yield(:page_title) %></title>
+  <link rel="preconnect" href="https://rsms.me/">
+  <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
   <link rel="stylesheet" type="text/css" href="stylesheets/style.css" data-turbo-track="reload">
   <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print">
   <link rel="stylesheet" type="text/css" href="stylesheets/highlight.css" data-turbo-track="reload">
-  <link href="images/favicon.ico" rel="icon" type="image/x-icon" />
+
+  <link rel="icon" href="images/favicon.ico" sizes="any">
+  <link rel="icon" href="images/icon.svg" type="image/svg+xml">
+  <link rel="apple-touch-icon" href="images/icon.png">
+
   <script src="javascripts/@hotwired--turbo.js" data-turbo-track="reload"></script>
   <script src="javascripts/clipboard.js" data-turbo-track="reload"></script>
   <script src="javascripts/guides.js" data-turbo-track="reload"></script>
+
   <meta property="og:title" content="<%= yield(:page_title) %>" />
   <meta name="description" content="<%= yield(:description) %>" />
   <meta property="og:description" content="<%= yield(:description) %>" />
   <meta property="og:locale" content="en_US" />
   <meta property="og:site_name" content="Ruby on Rails Guides" />
   <meta property="og:image" content="https://avatars.githubusercontent.com/u/4223" />
   <meta property="og:type" content="website" />
+
+  <meta name="theme-color" content="#C81418">
 </head>
+
 <body dir="<%= @direction %>" class="guide">
-  <% if badge_version = @edge ? "edge" : @version %>
-  <div>
-    <div id="version-badge"><%= badge_version %></div>
-  </div>
-  <% end %>
-  <div id="topNav">
+  <nav id="topNav" aria-label="Secondary">
     <div class="wrapper">
       <strong class="more-info-label">More at <a href="https://rubyonrails.org/">rubyonrails.org:</a> </strong>
       <span class="red-button more-info-button">
```

Comment:
Very minor: This is shown in smaller screens, but the cursor is text. It should instead be a cursor that indicates a  clickable element (even thought this is targeting more mobile screens). Hopefully this GIF explains it better:

![CleanShot 2024-03-18 at 17 39 34](https://github.com/rails/rails/assets/26328/b0fe614c-e4e1-4e39-ab49-88e750bae117)

Reply:
Ah straightforward fix. Will switch cursor to `pointer`

Reply:
Resolved in 3fb079a9